### PR TITLE
Fixed dump for cases when special URL characters in task name

### LINKED ISSUE #1161: Can't dump tasks with special characters in name (like: #)

### PR DESCRIPTION
**Pull request by bsekachev**
Resolved #1161

### CODE REVIEW

File: cvat-core/src/server-proxy.js [hunk 1]
```diff
@@ -569,7 +571,8 @@
                             if (response.status === 202) {
                                 setTimeout(request, 3000);
                             } else {
-                                url = `${url}&action=download`;
+                                query = `${query}&action=download`;
```
Comment (nglazov) [inline cvat-core/src/server-proxy.js]: Suggest to use some (library)[https://www.npmjs.com/package/qs], to work with query as object

**Comment by nmanovic** [comment]
@bsekachev , can we add a unit test?

File: cvat-core/src/server-proxy.js (hunk 1 above)
Comment (bsekachev) [reply cvat-core/src/server-proxy.js]: @nglazov 

Looks like overhead, to add library for such a minor case
What is more, similar functionality for working with query string is already part of native JS: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

**Comment by bsekachev** [comment]
@nmanovic 

Yes, but probably it will be in next PRs, just because we are short of time to implement different features that affect UI till next release.

File: cvat-core/src/server-proxy.js (hunk 1 above)
Comment (nglazov) [reply cvat-core/src/server-proxy.js]: Yes, it's good solution too
I mean, that better to work with objects instead of strings

Comment (bsekachev) [reply cvat-core/src/server-proxy.js]: It makes sense

**Pull request merged**